### DISCUSSION TITLE
How can I upload a local file to a Codespaces

### DISCUSSION THREAD

**Discussion by maziac** [community/community#30833] [Codespaces] [answered]
Hi,

I would like to get a local file from my PC to code spaces.
But directly, not via a git repository.

Is this possible at all?

**Comment by ihnatenkoo** [comment, marked as answer]
Hello, right click in the explorer view:

![ubIRj](https://user-images.githubusercontent.com/89586345/186739567-d9c92c26-a671-43ef-b8e1-2c0fcc061f92.png)

**Reply by DevOpsAIguru123** [reply]
I can not upload a folder

**Reply by lokman2k5** [reply]
> I can not upload a folder

duh

**Comment by maziac** [comment]
Thanks for the hint. That does it.
